pkg/driver/node/mounter/pod_unmounter.go

```diff
@@ -68,25 +65,82 @@ func (u *PodUnmounter) unmountSourceForPod(mpPod *corev1.Pod) {
 	unlockMountpointPod := lockMountpointPod(mpPod.Name)
 	defer unlockMountpointPod()
 
+	source := filepath.Join(SourceMountDir(u.kubeletPath), mpPod.Name)
+
+	// Check mountpoint status and handle special cases
+	isMountpoint, err := u.checkMountpointAndCleanup(source)
+	if err != nil {
+		klog.Errorf("Error during handling mountpoint check for %s: %v", source, err)
+		return
+	}
+	if !isMountpoint {
+		return
```

Comment:
Should we emit a log here?

Reply:
I originally had it, but then removed as it produced a lot of noise in logs because `HandleMountpointPodUpdate` is invoked around 4-5 times during unmount.

I can add it back with V(5) level? `klog.V(5).Infof()`

Reply:
Oh okay, I think it's fine to not add now then